src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchPhraseAPIIntegrationTest.kt

```diff
@@ -0,0 +1,185 @@
+package uk.gov.justice.hmpps.offendersearch.controllers
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.restassured.RestAssured
+import io.restassured.config.ObjectMapperConfig
+import io.restassured.config.RestAssuredConfig
+import org.elasticsearch.client.RestHighLevelClient
+import org.hamcrest.Matchers.equalTo
+import org.hamcrest.Matchers.notNullValue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.test.annotation.DirtiesContext
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.TestContext
+import org.springframework.test.context.TestExecutionListeners
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.context.support.AbstractTestExecutionListener
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener
+import uk.gov.justice.hmpps.offendersearch.dto.OffenderAlias
+import uk.gov.justice.hmpps.offendersearch.dto.OffenderDetail
+import uk.gov.justice.hmpps.offendersearch.dto.ProbationArea
+import uk.gov.justice.hmpps.offendersearch.dto.SearchPhraseFilter
+import uk.gov.justice.hmpps.offendersearch.util.JwtAuthenticationHelper
+import uk.gov.justice.hmpps.offendersearch.util.LocalStackHelper
+import java.lang.reflect.Type
+import java.time.LocalDate
+import java.util.*
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles(profiles = ["test", "localstack"])
+@ExtendWith(SpringExtension::class)
+@TestExecutionListeners(listeners = [DependencyInjectionTestExecutionListener::class, OffenderSearchPhraseAPIIntegrationTest::class])
+@ContextConfiguration
+@DirtiesContext
+class OffenderSearchPhraseAPIIntegrationTest : AbstractTestExecutionListener() {
```

Comment:
Can we pare this back to the minimal spring boot test config - not sure if all the complexity is needed?